test/orange/g4org/SolidConverter.test.cc

```diff
@@ -734,47 +734,76 @@ TEST_F(SolidConverterTest, polycone)
 
 TEST_F(SolidConverterTest, polyhedra)
 {
-    static double const z[] = {-0.6, 0.6};
-    static double const rmin[] = {0, 0};
-    static double const rmax[] = {61.85, 61.85};
+    static double const z[] = {-10, 0, 20, 25};
+    static double const no_rmin[] = {0, 0, 0, 0};
+    static double const rmin[] = {20, 4, 17, 4};
+    static double const rmax[] = {30, 10, 20, 5};
+    static real_type const sqrt2 = std::sqrt(2);
 
-    // Flat-top hexagon
+    // Full diamond shape, no interior
     this->build_and_test(
         G4Polyhedra(
-            "HGCalEEAbs", 330 * deg, 360 * deg, 6, std::size(z), z, rmin, rmax),
```

Comment:
Can you please keep but update these original tests, and add your new ones after? The HGCal one is an actual use case that we need to check, and I think I verified the full 360-degree polyhedra as well (see `test/geocel/data/polyhedra.gdml`):
<img width="3600" height="1200" alt="polyhedra" src="https://github.com/user-attachments/assets/63847beb-7dd9-4f10-9625-4e7fa4911a5d" />

Reply:
Yes, sorry. The polyhedra GDML solids (which have been verified with the ray tracer) are constructed as Geant4 solids in the tests that were deleted. So keeping them ensures we have a "validated" test case.